accepted/0000-sbom-command.md

```diff
@@ -162,7 +162,7 @@ _Recommendation: Add a distinct command for generating an SBOM._
 
 * Does `npm-sbom` command have a notion of a “default” SBOM format? Do we give preference to one of CycloneDX/SPDX or do we remain totally neutral (possibly at the expense of DX)? \
  \
-_Recommendation: Default to SPDX if no format is specified._
+_Recommendation: Remain neutral with regard to SPDX vs CycloneDX. Make the `--sbom-format` flag mandatory.
```

Comment:
npm should still be listing the options, at which point, why not default to all of them? being neutral would encourage users to support everything until the ecosystem (maybe) picks a winner.